book/animations.md

```diff
@@ -218,10 +218,15 @@ and draw a given page can be quite complex.
 
 [gpu]: https://en.wikipedia.org/wiki/Graphics_processing_unit
 
+::: {.installation}
+
 First, we'll need to install the OpenGL library:
 
     pip3 install PyOpenGL
 
+
+:::
```

Comment:
Not sure I agree with this one; we usually use an `installation` box for optional steps that one some users will need.

Reply:
Shouldn't "installation" be for all software installation instructions?

Reply:
Well, we've never really defined these boxes well, but I think installation instructions are normally part of the main part of the text, with quirks described elsewhere. Rationalizing the boxes would be good, but maybe that's a separate PR—we'd want to look over how installation instructions are treated in other chapters. (Also maybe `quirk` and `installation` boxes are in fact the same thing, if they both describe installation quirks?)